src/main/java/pl/akai/bookcrossing/list/BookRestController.java

```diff
@@ -0,0 +1,39 @@
+package pl.akai.bookcrossing.list;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import pl.akai.bookcrossing.model.Book;
+
+@RestController
+@RequiredArgsConstructor
+public class BookRestController {
+
+    private final BookBean bookBean;
+    private final BookDao bookDao;
+
+    @RequestMapping("/book/{id}/rent")
+    @ResponseStatus(HttpStatus.OK)
+    public void bookRental(@PathVariable(name = "id") Integer bookId) {
+        bookBean.insertBookUserRequest(bookId);
+    }
+
+    @RequestMapping("/my-books/available-negation")
+    @ResponseStatus(HttpStatus.OK)
+    public void bookIsAvailableNegation(@RequestParam(value = "id") Integer bookId) {
+        Book book = bookDao.getBookById(bookId);
+        bookBean.updateIsAvailable(bookId, !book.isAvailable());
+    }
```

Comment:
Nie wiem czy dobrze zrozumiałem tą uwagę odnośnie ustawiania IsAvailable, ale jeżeli miałaby się wywoływać na "zakończenie wypożyczenia" to mógłbym jeszcze dodać usunięcie aktualnego czytelnika